### ISSUE TITLE
Make copied tags editable again after breaking the upstream link to library content

### ISSUE THREAD

**Issue by bradenmacdonald** [openedx/modular-learning#244]
As a course author, when I import library content into a course (whether via copy-paste or "Add Library Content..."), it will show the  assigned tags as read-only. If I then break the link to the upstream library, I should be able to delete those tags.

This is a follow-up to https://github.com/openedx/modular-learning/issues/177

**Comment by pomegranited** [comment]
@bradenmacdonald or @navinkarkera Are there any other ways to "unlink" a downstream block other than deleting its upstream block?

**Comment by bradenmacdonald** [comment]
@pomegranited I don't think we expose any way to "unlink" a course block from the library version in the UI. On the backend it's as simple as removing the "upstream_" attribute(s) from the OLX.

@marcotuts @jmakowski1123 @sdaitzman Do you think we want to have an "unlink from library" button in the UI in the future? I just checked [the relevant Figma design (?)](https://www.figma.com/design/Y3gSyrpG6uncUUfjK7eQuo/Content-Libraries-v2.1-%5BOpen-edX%5D?node-id=1558-50801&t=wnmqFdvhcRdEwg28-4) but didn't see anything like that.

**Comment by pomegranited** [comment]
@jmakowski1123 @lizc577 @sdaitzman @marcotuts This is ready for AC testing on the sandbox.

1. Create a block in a content library, and add some tags to it.
1. Publish the block.
1. Copy the library block into a course, either using the staged content copy/paste, or by adding a Library Content block, and selecting your published tagged block.
   Should see the tags copied from the library block on the new course block, and they should _not_ be deleteable.
1. Add some more tags to the new course block.
   These added tags should be deletable.
   Note: something is broken on the MFE Unit page and it won't let me add/edit tags, so I've enabled the legacy Studio edit page instead, cf [contentstore.new_studio_mfe.use_new_unit_page](https://tagging-preview.staging.do.opencraft.hosting/admin/waffle/flag/37/change/)
1. Go back to your content library, and delete the tagged block.
1. Return to the course block, and re-open the "manage tags" sidebar.
    All the tags should remain, and all are deletable now.

**Comment by edschema** [comment]
Working well, thanks!
